array-manipulation-1/src/last.js

```diff
@@ -0,0 +1,8 @@
+/* exported last */
+function last(array) {
+  let lastElement;
+  if (array[array.length - 1] !== undefined) {
+    lastElement = array[array.length - 1];
+  }
+  return lastElement;
+}
```

Comment:
You can also just return the last index directly. We don't care if it's undefined or not.
`return array[array.length - 1]`